rust-bot/src/main.rs

```diff
@@ -15,12 +15,15 @@ use serenity::{
         guild::Role,
         id::RoleId,
         permissions::Permissions,
+        prelude::MessageActivityKind,
     },
     prelude::*,
     utils::MessageBuilder,
     Error,
 };
 
+const CHANNEL__LISTENING_PARTY: u64 = 742445700998103132;
```

Comment:
I dig it. Do we need to stuff a bunch of these (hard coded ids) somewhere?

Reply:
yeah probably, any time we need something. We can probably throw this into a module at some point with the other consts?